runtime/src/prioritization_fee_cache.rs

```diff
@@ -407,6 +407,32 @@ impl PrioritizationFeeCache {
             .flatten()
             .collect()
     }
+
+    pub fn get_prioritization_fees_with_accounts(&self, account_keys: &[Pubkey]) -> Vec<u64> {
+        self.cache
+            .read()
+            .unwrap()
+            .iter()
+            .filter_map(|(_slot, prioritization_fee)| {
+                let prioritization_fee = prioritization_fee.entry();
+                let prioritization_fee_read = prioritization_fee.lock().unwrap();
+                prioritization_fee_read.is_finalized().then(|| {
+                    let mut fee = prioritization_fee_read
+                        .get_min_transaction_fee()
+                        .unwrap_or_default();
```

Comment:
@taozhu-chicago here's one.
It seems right to me to return some value for all slots, even if no prioritization fee is required. Does this jive with you?

Reply:
Makes sense to me. If a slot didn't require prioritization fee, it is same as required `0` fee.